### PR TITLE
Explanation text

### PR DESCRIPTION
**Pull request by Lokymil**

### CODE REVIEW

File: src/components/interview/Theory.js [hunk 1]
```diff
@@ -31,30 +31,42 @@ const Home = () => {
...
-          tincidunt ex odio, ut convallis nisi venenatis non. Sed sodales vitae
-          augue vitae convallis. Cras nunc velit, vehicula vitae auctor id,
-          convallis eget lorem. Curabitur eu rhoncus urna. Etiam eu vestibulum
-          elit. Donec hendrerit ligula nec leo laoreet commodo ut et quam.
-          Suspendisse viverra placerat leo in dignissim.
+          Understand and help your team to grow is a difficult task. The point
+          of this website is to generate factual and mesurable data base on your
+          co-worker answer.
+          <br />
+          These dat will help you understand you team and take decision to make
```
Comment (saramorillon) [inline src/components/interview/Theory.js]: "These data will help you understand your team"

File: src/components/interview/Theory.js [hunk 2]
```diff
@@ -31,30 +31,42 @@ const Home = () => {
...
       <section id="theory--how">
         <h2>How does it work?</h2>
         <div className="with-image">
-          <img src="/img/question-mark.svg" alt="" />
+          <img className="icon" src="/img/question-mark.svg" alt="" />
           <p>
             First of all, you will ask several questions to your collaborator.
-            These questions will be used to determine ...
+            These questions will be used to determine their general mindset.
+            <br /> Are they feeling qualified for the job have to do and are
```
Comment (saramorillon) [inline src/components/interview/Theory.js]: "Are they feeling qualified for the job they have to do"

File: src/components/interview/Theory.js [hunk 3]
```diff
@@ -31,30 +31,42 @@ const Home = () => {
...
         </div>
         <div className="with-image">
-          <img src="/img/cards.svg" alt="" />
+          <img className="icon" src="/img/cards.svg" alt="" />
           <p>
-            Then, your collaborator will be asked to use Moving Motivator cards
-            to sort and assess theirs inherent motivation. With these
-            information, you will be able to...
+            Then, your collaborator will be asked to use Moving Motivator game
+            to sort and assess theirs inherent motivation.
```
Comment (saramorillon) [inline src/components/interview/Theory.js]: "their inherent motivations"

File: src/components/interview/Theory.js [hunk 4]
```diff
@@ -31,30 +31,42 @@ const Home = () => {
...
+            <br />
+            With these information, you will be able to understand what each of
+            your co-worker are motivated by and what do they expect to find in
```
Comment (saramorillon) [inline src/components/interview/Theory.js]: "what do they expect" or "what they expect" ?

**Review by saramorillon — CHANGES_REQUESTED** [review]

File: src/components/interview/Theory.js (hunk 1 above)
Comment (Lokymil) [reply src/components/interview/Theory.js]: fix

File: src/components/interview/Theory.js (hunk 2 above)
Comment (Lokymil) [reply src/components/interview/Theory.js]: done

File: src/components/interview/Theory.js (hunk 3 above)
Comment (Lokymil) [reply src/components/interview/Theory.js]: done

File: src/components/interview/Theory.js (hunk 4 above)
Comment (Lokymil) [reply src/components/interview/Theory.js]: done

**Pull request merged**